transform/_procesador_i3.py

```
@@ -0,0 +1,455 @@
+            diario_raw['fecha'] = pd.to_datetime(diario_raw['fecha'])
+            diario_filtered = diario_raw[diario_raw['fecha'].dt.date == target_date].copy()
+            if diario_filtered.empty:
+                return pd.DataFrame()
+
+            diario_config = DiarioConfig()
+            diario_processed = self._apply_market_filters_and_id(diario_filtered, diario_config)
+            diario_processed = self._standardize_datetime(diario_processed, dataset_type)
+            diario_processed = self._select_and_finalize_columns(diario_processed, dataset_type)
+            return diario_processed
+        except Exception as e:
+            raise ValueError(f"Error loading diario data for intra baseline: {e}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)